watchdog/readconfig.go

```diff
@@ -49,21 +52,13 @@ func (ReadConfig) Read(hasEnv HasEnv) WatchdogConfig {
 
 	cfg.faasProcess = hasEnv.Getenv("fprocess")
 
-	readTimeout := parseIntValue(hasEnv.Getenv("read_timeout"))
-	writeTimeout := parseIntValue(hasEnv.Getenv("write_timeout"))
+	readTimeout := parseIntOrDurationValue(hasEnv.Getenv("read_timeout"), time.Second*5)
```

Comment:
These variables aren't used anywhere else other than in the assignment to the cfg, we should probably just parse and assign directly to `cfg.readTimeout` and `cfg.writeTimeout`.  Other than that, everything looks great.